processing/src/main/java/io/druid/segment/data/TmpFileIOPeon.java

```diff
@@ -34,8 +34,19 @@
 */
 public class TmpFileIOPeon implements IOPeon
 {
+  boolean allowConflict;
```

Comment:
can we make this a private final? does allowConflict=true mean we can overwrite the tmp file? If so, can we call this isOverrideAllowed or something?